src/main.ts

```diff
@@ -5,6 +5,7 @@ import App from './App.vue'
 import router from './router'
 
 import './assets/main.css'
+import 'cooperation_jn_common_ui/dist/style.css'
```

Comment:
pardon my lack of knowledge. Will this work when we don't have the local `cooperation_jn_common_ui` (use from the GitHub repository directly)